mlflow/models/evaluation/base.py

```diff
@@ -212,6 +382,21 @@ def eval_fn(
         genai_metric_args: (Optional) A dictionary containing arguments specified by users
             when calling make_genai_metric or make_genai_metric_from_prompt. Those args
             are persisted so that we can deserialize the same metric object later.
+        require_strict_signature: (Optional) Whether the eval_fn needs to follow a strict signature.
```

Comment:
Do users need to touch this argument?

Reply:
No, I can add a note for this!

Reply:
Wait I think users can use this parameter, if they want to manually create some custom metric using `make_metric`. Setting it to false doesn't break anything though, the metric is still callable.

Reply:
Do we really need to expose this parameter?

Reply:
Emm you mean we remove this parameter in the docstring?

Reply:
yeah if there is no solid use case.